src/components/admin/ImageController.jsx

```diff
@@ -138,6 +148,14 @@ export default class ImageUploader extends BaseComponent {
       this.addImagePreviewUrl(file);
     });
 
+    if (tooLargeFileNames.length !== 0) {
+      let names = '\n';
+      tooLargeFileNames.forEach((file) => {
```

Comment:
This works, good job! But there's a smoother way we can do it, like this:

```js
const alertMessage = tooLargeFileNames.reduce((currentMessage, filename) => {
  return `${currentMessage}\n${file.name} (${sizeCalculation) KB)`;
}, 'The following files exceeded the maximum upload size:\n');
```

See [these docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) for an explanation of the reduce function